This extension adds the ability to split a cell vertically in a notebook.

cdknorow opened the pull request:
This shows the idea, in this extension you can currently only toggle the split cell using shift-s in command mode. I haven't added a button yet. The information is stored in the cell metadata. 


<img width="1280" alt="3f97874c-f93d-11e5-9f3c-7efe97d7dac3" src="https://cloud.githubusercontent.com/assets/884424/14452701/419742b2-0045-11e6-8960-71a3aeaf7b26.png">

jcb91 commented:
Looks nice! Could you use Jupyter rather than IPython for the namespace please?

For adding a toolbar button, it's probably worth making the switching an action so that it can also be used more generally (see, e.g. [nbextensions/usability/init_cell/main.js#L40-L50](https://github.com/ipython-contrib/IPython-notebook-extensions/blob/22dd78c42ec8dd1724120345b7472d595bfdb264/nbextensions/usability/init_cell/main.js#L40-L50) for an example).

cdknorow commented:
@jcb91 Thanks for the advice. I updated the name space and added a nice button. I messed up the commit update, ended up just creating a new pull requests instead.
